src/App.jsx

```diff
@@ -1,15 +1,27 @@
 import { Route, Routes } from "react-router-dom";
 
-import Landing from "./views/landing";
-import Login from "./views/login";
-import Signup from "./views/signup";
+import Landing from "./views/Landing";
+import Login from "./views/Login";
+import Signup from "./views/Signup";
+import Dashboard from "./views/Dashboard";
+import PrivateRoutes from "./views/PrivateRoutes";
+import AllPosts from "./views/AllPosts";
+import Create from "./views/Create";
 
```

Comment:
Landing, Login, Signup have all been changed to starting with a capital in the app.jsx file. It does not look like they have for the actual file name. When I initially pulled the code down I was getting errors from this. Please change the file names to all start with a capital letter